Item.h

```diff
@@ -5,13 +5,13 @@
 struct Character;
 struct Item
 {
-    virtual ~Item() { }
-    virtual void use(Character*) = 0;
     Item(const std::string& name_, int effect ) : name(name_), boost(effect) { }
-    
+    virtual ~Item() { };
```

Comment:
Do not randomly add semicolons after function definitions.
Semicolons are used to indicate the end of an expression.
they are also used at the end of UDT declarations/definitions: `struct Foo { };`
- [ ] revise